### PR TITLE
Eassy proposal: devops and agile

### PR DESCRIPTION
**Pull request by Linnea55**
# Assignment Proposal

## Title

Working with DevOps and Agile together

## Names and KTH ID
  - Linnea Hagman (lhagm@kth.se)
 
## Deadline

Task 4

## Category

Essay

## Description

Planned headings (wording and order might change):

Introduction
What is Agile?
What is DevOps?
How do they complement each other (to get a better quality of service delivered)?
How can agile and devops be used successfully together?
What are the challenges of using Devops and agile together? 
Conclusion

### CODE REVIEW

**Comment by cesarsotovalero** [comment]
Hi @Linnea55,
Thanks for submitting this proposal.
I don't think Agile practices are **comparable** to DevOps.
I suggest you to the essay on the **complementarity** of both practices.
Please update the description with more details regarding the essay planning (topic and sections).
I'll revise your PR once you update the README.

File: contributions/essay/lhagm/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Task 4
+
+## Category
+
+Essay
+
+## Description
+
+I will erite an essay about agile vs devops. What are the similarities and differences? Can they complement eachother? Should you prefer one over the other?
```
Comment (cesarsotovalero) [inline contributions/essay/lhagm/README.md]: Elaborate more on the complementarity of Agile and DevOps practices. Include a Headline of the topics that you plan to cover in the essay.

**Review by cesarsotovalero — CHANGES_REQUESTED** [review]

**Comment by Linnea55** [comment]
Hi @cesarsotovalero,

yesterday I made some changes in the README.md file. Hopefully they are sufficient to merge this PR.

**Review by cesarsotovalero — APPROVED** [review]

**Comment by cesarsotovalero** [comment]
Hi @Linnea55, LGTM. 
Good luck!

**Pull request merged**